jetstream/configuration_mgmt/terraform.md

```diff
@@ -66,3 +54,18 @@ output "ORDERS_SUBJECTS" {
 }
 ```
 
+### Manual installation (obsolete)
```

Comment:
we can drop all this

Reply:
Remove the manual instructions only?

Reply:
yup, its been long enough that registry is in place that everyone should use that now

Reply:
dropped